contracts/LockedTokens.cdc

```diff
@@ -291,7 +292,7 @@ pub contract LockedTokens {
 
         /// Returns the locked account balance for this token holder.
```

Comment:
should we mention in the comments what this does?

Reply:
Yes good idea. I'll update the comment